lib/reporter/reporters/html.js

```diff
@@ -122,15 +122,19 @@ class HtmlReporter extends BaseReporter {
   }
 
   aggregateStats() {
-    const startTime = new Date(this.results.startTimestamp).getTime();
-    const endTime = new Date(this.results.endTimestamp).getTime();
```

Comment:
@Vaibhavsahu2810 Thanks for this PR. But, were you able to figure out why the subtraction of the above two variables is giving the wrong time?

Ideally, we should try to find the root cause of the problem and try to tackle that instead of just introducing a fix. If the `this.results.endTimestamp` is wrong above, we should fix that because it might be causing problems at other places as well.

Also, if the tests are running in parallel, we wouldn't just want to add up the time of all the tests but instead show the time for the which whole test suite actually ran (if I'm not missing something here?).

Reply:
@Vaibhavsahu2810 That's fine. We are supposed to show the overall time (including setup/teardown and the time spent in between the test runs) in the Overall Performance section. This section should basically show the total run time of the Nightwatch process and not the sum of the run-time of individual test cases.

Let's say you run three tests in parallel using Nightwatch and the individual run time of the tests are as follows:
* Test 1: 30 secs
* Test 2: 75 secs
* Test 3: 40 secs

Then the total run time of the Nightwatch process need not be same as the sum of the run time of the three test cases (since the tests are running in parallel) but it should always be greater then the run time of the longest running test (75 secs).

This is exactly the problem in the issue where the overall performance shows a time less than the time of the longest running test, which is not possible.